### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by puppet101** [itsyoavshalev/Image-Animation-with-Perturbed-Masks#1]
Hi, thanks for sharing the code， could please upload the pretrained model in order to have a test?

**Comment by ahuirecome** [comment]
I trained the model for several days, but couldn't get any good result.

**Comment by itsyoavshalev** [comment]
@ahuirecome  please verify that you are using the same versions introduced in README.md (specifically the pytorch version).

**Comment by itsyoavshalev** [comment]
@puppet101 pretrained models will be added soon.

**Comment by MrYANG23** [comment]
> I trained the model for several days, but couldn't get any good result.

hi can you show your trained results ,or Wechat communication
